daemon/cmd/kube_proxy_replacement.go

```diff
@@ -710,6 +712,30 @@ func detectNodeDevice(ifidxByAddr map[string]int) (string, error) {
 	return link.Attrs().Name, nil
 }
 
+// expandDevices expands all wildcard device names to concrete devices.
+// e.g. device "eth+" expands to "eth0,eth1" etc. Non-matching wildcards are ignored.
+func expandDevices() {
+	allLinks, err := netlink.LinkList()
+	if err != nil {
+		log.WithError(err).Fatal("Cannot list network devices via netlink")
+	}
+	var expandedDevices []string
```

Comment:
I'd use map (e.g. `map[string]struct{}`) instead to avoid duplicates.

Reply:
Added the deduplication and tests for it.